.laminas-ci.json

```diff
@@ -0,0 +1,5 @@
+{
+    "ignore_php_platform_requirements": {
+        "8.1": true
```

Comment:
This shouldn't apply anymore :| 

I think we need to bump to `laminas/laminas-coding-standard: ~2.3.0`, but otherwise, `8.1` used to install cleanly, before this change.